### PR TITLE
fix MainTemplate.getAssetPath deprecation.

### PR DESCRIPTION
**Pull request by ivankoop**

### CODE REVIEW

**Comment by jantimon** [comment]
Looks good 👍 thanks a lot - right now webpack 5 has only limited support for child compilers so we will need to wait until this issue 5 is ready : https://github.com/webpack/webpack/issues/11909

File: src/compiler.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ module.exports.run = (faviconOptions, context, compilation) => {
       // Replace [hash] placeholders in filename
       const result = extractAssetFromCompilation(
         compilation,
-        compilation.mainTemplate.getAssetPath(filename, { hash, chunk })
```
Comment (inoyakaigor) [inline src/compiler.js:66]: There is one more place which needs tobe changed https://github.com/jantimon/favicons-webpack-plugin/blob/ed6c0217e349675d09ab456139b96918ef6d727a/src/index.js#L182

_inb4 I'm not maintainer, just researching before made my own PR 🤷🏻‍♂️_

**Comment by drolsen** [comment]
> Looks good  thanks a lot - right now webpack 5 has only limited support for child compilers so we will need to wait until this issue 5 is ready : [webpack/webpack#11909](https://github.com/webpack/webpack/issues/11909)

https://github.com/webpack/webpack/issues/11909#issuecomment-734161968

It appears you are actually able to get around the child compiler limitation now and have a Webpack 6 version, can we get some kind of update around what might be holding this up?

Thanks!

**Comment by jantimon** [comment]
released in 5.0.0

**Pull request closed without merging**